components/tools/OmeroPy/src/omero/plugins/fs.py

```diff
@@ -840,7 +840,10 @@ def _usage_obj(self, obj):
                 else:
                     ids = parts[1].split(",")
                     ids = map(long, ids)
-                    objects[klass] = ids
+                    if klass in objects:
+                        objects[klass].extend(ids)
+                    else:
+                        objects[klass] = ids
             except:
```

Comment:
The above four lines can be shortened to `objects.setdefault(klass, []).extend(ids)`